Rakefile

```diff
@@ -12,6 +12,15 @@ RSpec::Core::RakeTask.new(:hpack) do |t|
   t.pattern = './spec/hpack_test_spec.rb'
 end
 
+task :h2spec do
+  system 'ruby example/server.rb -p 9000 &', out: File::NULL
+  sleep 1
+
+  system 'spec/h2spec/h2spec.darwin -p 9000'
```

Comment:
At a minimum, we should provide a comment for where to download the binary?

We can also exit early with an error for non-darwin:```if /darwin/ !~ RUBY_PLATFORM) ... ```